src/components/shared/Fields/Checkbox/HookForm/Checkbox.css

```diff
@@ -0,0 +1,63 @@
+@value checkmark: "data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' version='1.1' viewBox='0 0 30 30'%3E%3Cg transform='translate(0,-350.38584)' fill='%2368D2DF'%3E%3Cpath d='M 26.558594,355.88672 11.042969,372.00195 3.4414062,364.10547 2,365.49219 11.042969,374.88477 28,357.27344 Z' /%3E%3C/g%3E%3C/svg%3E";
```

Comment:
I think this is a better approach than the approach in the other Checkbox, which is drawing a checkbox using the border of a span element.

However, ideally the svg would live in the icons folder and be imported in here using `url(~images/icons/check-mark.svg)`. Webpack didn't like my attempts so I'm using this workaround for now. Anyone know how to appease webpack here?

Reply:
Does the svg loader know to parse css files as well ?